grouper/services/permission.py

```diff
@@ -27,6 +27,16 @@ def __init__(self, audit_log, permission_repository, permission_grant_repository
         self.permission_repository = permission_repository
         self.permission_grant_repository = permission_grant_repository
 
+    def create_permission(self, name, description=""):
+        # type: (str, str) -> None
+        self.permission_repository.create_permission(name, description)
+
+    def create_system_permissions(self):
```

Comment:
Same question as on the `initialize_X_group`s above (with probably the same answer 😄)